cmd/cored/loopback_auth.go

```diff
@@ -2,17 +2,8 @@
 
 package main
 
-import (
-	"chain/core/config"
-	"net"
-	"net/http"
-)
+import "chain/core/config"
 
 func init() {
 	config.BuildConfig.LoopbackAuth = true
-	loopbackAuth = func(req *http.Request) bool {
```

Comment:
Do you want to drop a comment in here pointing the reader to where the logic is? Like

```go
// See $CHAIN/net/authz/loopback_authz.go for the implementation.
```